packages/create-amplify/src/get_project_root.test.ts

```diff
@@ -72,6 +74,48 @@ void describe('getProjectRoot', () => {
     assert.equal(projectRoot, path.resolve(userInput));
   });
 
+  void it('prints warning if creation of project root failed and path is absolute', async () => {
+    process.env.npm_config_yes = 'false';
+    const userInput = 'some/absolute/path';
+    mock.method(AmplifyPrompter, 'input', () => Promise.resolve(userInput));
+    const expectedError = new AmplifyUserError('ProjectDirectoryCreateError', {
+      message: `Failed to create project directory`,
+      resolution: `Ensure that ${userInput} is the correct path and you have write permissions to this location.`,
+    });
+    fsMkDirSyncMock.mock.mockImplementationOnce(() =>
+      Promise.reject(expectedError)
+    );
+    await assert.rejects(
+      () => getProjectRoot(),
+      (err: Error) => {
+        const sampleStderr = `some random stderr
+before the actual error message
+${util.inspect(err, { depth: null })}
+and some after the error message`;
+
+        const actual = AmplifyError.fromStderr(sampleStderr);
+        assert.deepStrictEqual(actual?.name, expectedError.name);
+        assert.deepStrictEqual(
+          actual?.classification,
+          expectedError.classification
+        );
+        assert.deepStrictEqual(actual?.message, expectedError.message);
+        assert.deepStrictEqual(actual?.details, expectedError.details);
+        assert.deepStrictEqual(actual?.cause?.name, expectedError.cause?.name);
+        assert.deepStrictEqual(
+          actual?.cause?.message,
+          expectedError.cause?.message
+        );
+        return true;
```

Comment:
You don't need all this. Just assert on the `err` that the `name`, `message` and `resolution` is correct.

Reply:
try change!

Reply:
@Amplifiyer 
thank you feedback!!!!!
I try change test case